src/actions/src.cr

```diff
@@ -1,19 +1,96 @@
+require "dppm/prefix/src"
+
 module DppmRestApi::Actions::Src
   extend self
   include RouteHelpers
 
+  struct MultiSet(K, V)
+    @data = {} of K => Set(V)
+
+    def add(at k : K, v : V) : Nil
+      @data[k] = Set{v} unless @data[k]?.try { |set| set << v }
+    end
+
+    def add(at k : K, v : Enumerable(V))
+      if @data[k]?
+        @data[k] &= v.to_set
+      else
+        @data[k] = Set(V).new v
+      end
+    end
+
+    forward_missing_to @data
+  end
+
+  struct ::DPPM::Prefix::PkgFile
```

Comment:
This isn't only needed for `src`, but also `pkg` and `app`.
Creating a package copy the source one, and creating an app may copy a package if self-contained.